### DISCUSSION TITLE
Netzwerk Phänomen

### DISCUSSION THREAD

**Discussion by ferdlgogosch** [hoylabs/OpenDTU-OnBattery#1055] [General]
Hallo, mir  ist seit Wochen ein Phänomen aufgefallen das ich erst heute nachstellen könnte.
Seit ich auf esp32-s3-16/8 umgestiegen bin hatte ich immer wieder (fast täglich meistens Nachmittags) WiFi Ausfälle. Manchmal länger über Stunden dann wieder kürzer einige Minuten. Ab heute weiß ich wie es zustande kommt: wenn ich meinen Echo Show 8 Internetradio in einer Gruppe (Wohnzimmer/Büro) starte wird das WiFi Signal vom esp32 gestört somit keine Übertragung in die Webseite und natürlich auch keine Übertragung in homeassistent! Ich habe es heute unzählige Male getestet sobald der stream gestartet wird ist die "Störung" da und in dem Moment wo der Stream beendet wir ist die Übertragung schlagartig wieder da.
Ich habe überlegt ob ich das berichteten sollte aber dachte wenn vielleicht irgendwann ein solches Phänomen auftaucht gibt es zumindest schon einmal einen Hinweis.
Nur noch zur Vollständigkeit, wenn der Stream "nur" auf dem Echo Show läuft, funktioniert alles ohne Probleme.
Schöne Grüße

**Comment by schlimmchen** [comment]
Was sagt denn deine OpenDTU-OnBattery über die Qualität des Netzes unter Info -> Netzwerk?

**Reply by ferdlgogosch** [reply]
Ein Screenshot 
![Screenshot_2024-06-17-21-14-59-38_df198e732186825c8df26e3c5a10d7cd](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/112643833/075ce1d8-0d09-4673-b639-d92114d278a7)

**Reply by ferdlgogosch** [reply]
Soll ich Mal einen Screenshot machen wenn die Echo Gruppe läuft?

**Reply by schlimmchen** [reply]
Ich denke das geht nicht, weil du die OpenDTU-OnBattery dann gar nicht mehr erreichst?

Jedenfalls negativ beeindruckend, dass diese Echos das Medium so verpesten, dass deine OpenDTU-OnBattery dann nichts mehr hört?!

Weiß Google dazu etwas?

**Reply by ferdlgogosch** [reply]
Habe es gerade probiert wenn ich lange genug warte kommt schon Mal eine Seite Zustande und dann ist das Signal natürlich in Ordnung. 
Ich habe immer wieder die IP Adresse angepingt und da sind mit unter schon Zeiten von 4 bis 6 Sekunden vorgekommen.
Da ich es erst  heute dem Echo Show zuordnen kann habe ich sonst nichts unternommen als es der OpenDTUOnBattery Gemeinde zu berichten.

**Comment by spcqike** [comment]
Was hast du denn als wifi gegenstelle? Einen Router? Einen accesspoint? Kannst du vielleicht hat dort mehr auslesen? Stören die Echos  dein 2.4GHz wlan weil sie es „verstopfen“, oder weil sie EMV Probleme haben? Bei letzterem wäre ggf. auch dein Nachbar betroffen und auch die Kommunikation zum wechselrichter, wenn du einen HM hast der das NRF24L01 benötigt.

**Reply by ferdlgogosch** [reply]
Vorab ich bin ein typischer User der zwar mit allen Anleitungen von OpenDTUOnBattery zurecht kommt (zusammenbau installieren etc.) aber keine Ahnung von Programmeieren hat. 
Es ist einen Router. Also ich habe einen hm300. 
Erst seit ich den esp32-S3 verwende ist das WLAN gestört wenn ich die Echo Gruppe starte. Vorher verwendete ich einen esp32 da gab es das Problem nicht. Betrifft aber nur die OpenDTUOnBattery, sonst sind keine Störungen im WLAN.

**Reply by ferdlgogosch** [reply]
Die Verbindung zwischen opendtu und hm300 ist nicht gestört. Also die verrichten ihre Arbeit wie eingestellt.
Es ist nur störend wenn man Einstellungen bzw Änderungen vornehmen will und dann nicht zugreifen kann.

**Comment by Solarteur** [comment]
Auch von mir vorab: Ich bin ein typischer User, der zwar mit allen Anleitungen von OpenDTU-OnBattery - früher oder später - zurechtkommt, aber keine Ahnung vom Programmieren hat.
@ferdlgogosch Hast du die Möglichkeit, ein anderes ESP32-S3 -16/8 - Board zu testen? Diese Boards sind Massenware und könnten (im "WLAN-Bereich") durchaus einen Defekt haben.

**Reply by ferdlgogosch** [reply]
Leider habe ich keines in Reserve aber ich war knapp daran ein neues Board zu bestellen... Jetzt wo ich es eingegrenzt habe kann ich damit erstmal leben. Aber wie ich mich kenne werde ich mir baldig ein neues Board organisieren und es gegen das jetztige tauschen. Ich werde hier berichten ob es dann funktioniert.

**Comment by ferdlgogosch** [comment]
Hallo, ich habe heute das neue esp32-s3-16/8  Board erhalten und jetzt gerade aufgesetzt (vielen Dank für den web-flasher...traumhaft 🥳). Es hängt noch am PC und ich habe erstmal einen IP-Adresse vergeben und gleich einen "dauer-Ping" laufen lassen. Echo Show 8 😃 in einer Gruppe WZ/büro  und danach WZ/Küche gestartet. Leider das selbe Problem; beim Internetradio Stream wird das neue sowie das "alte" Board gestört, so das es auf der Webseite bzw im homeassistent nicht erreichbar ist.
Vielleicht ist es ein Einzel Schicksal, aber heute ist ja nicht aller Tage Abend und mir wird schon noch was einfallen. Schönen Gruß
